### DISCUSSION TITLE
Easiest way to draw (or magnet) dropped items to player's position??

### DISCUSSION THREAD

**Discussion by SirEnder125** [FabricMC/fabric-api#2123] [Mod Dev Support] [answered]
Hey again! I just wanted to know if there is a way to magnet dropped items to the player? I want to make an item, preferably handheld, that will act as a magnet, so that the player can pick up items from far away. I would usually provide some code, but as I am totally bewildered as to how to reach this end, I haven't written any up to this point. 😥  Sorry if this is the wrong place to ask. 😵‍💫

**Comment by warjort** [comment, marked as answer]
https://www.curseforge.com/minecraft/mc-mods/rf-magnet-fabric

**Reply by SirEnder125** [reply]
Aight. Looks pretty simple actually. Thanks!

 Although in [MagnetItem.java](https://github.com/Huys-Arthur/E-Magnet-FABRIC/blob/main/src/main/java/com/rangetuur/emagnet/items/MagnetItem.java), is the "energy" part really important? I don't see it used in the actual magnetting.

[EDIT] nevermind, I figured out that that part of the code is to make the magnet battery-powered.
